openff/interchange/_tests/unit_tests/interop/openmm/test_hmr.py

```diff
@@ -80,6 +81,57 @@ def test_mass_is_positive(sage):
         sage.create_interchange(Molecule.from_smiles("C").to_topology()).to_openmm(hydrogen_mass=5.0)
 
 
-def test_virtual_sites_unsupported(tip4p, water):
-    with pytest.raises(UnsupportedExportError, match="with virtual sites"):
-        tip4p.create_interchange(water.to_topology()).to_openmm(hydrogen_mass=2.0)
+@pytest.mark.parametrize("hydrogen_mass", [1.1, 3.0])
+def test_hmr_not_applied_to_tip4p(
+    tip4p,
+    water,
+    hydrogen_mass,
+):
+    pytest.importorskip("openmm")
+
+    system = tip4p.create_interchange(
+        Topology.from_molecules(2 * [water]),
+    ).to_openmm_system(
+        hydrogen_mass=hydrogen_mass,
+    )
+
+    masses = [system.getParticleMass(particle_index)._value for particle_index in range(system.getNumParticles())]
+
+    assert masses[0] == masses[3] == 15.99943
+    assert masses[1] == masses[2] == masses[4] == masses[5] == 1.007947
+    assert masses[6] == masses[7] == 0.0
+
+
+@pytest.mark.parametrize("hydrogen_mass", [1.1, 3.0])
+def test_hmr_with_ligand_virtual_sites(sage_with_bond_charge, hydrogen_mass):
+    """Test that a single-molecule sigma hole example runs"""
+    pytest.importorskip("openmm")
+    import openmm
+    import openmm.unit
+
+    molecule = MoleculeWithConformer.from_mapped_smiles("[H:5][C:2]([H:6])([C:3]([H:7])([H:8])[Cl:4])[Cl:1]")
+
+    # should work fine with 4 fs, but be conservative and just use 3 fs
+    simulation = sage_with_bond_charge.create_interchange(molecule.to_topology()).to_openmm_simulation(
+        integrator=openmm.VerletIntegrator(3.0 * openmm.unit.femtosecond),
+        hydrogen_mass=hydrogen_mass,
+    )
+
+    system = simulation.system
+
+    assert system.getParticleMass(0)._value == 35.4532
+    assert system.getParticleMass(3)._value == 35.4532
+
+    assert system.getParticleMass(1)._value == 12.01078 - 2 * (hydrogen_mass - 1.007947)
+    assert system.getParticleMass(2)._value == 12.01078 - 2 * (hydrogen_mass - 1.007947)
+
+    assert system.getParticleMass(4)._value == hydrogen_mass
+    assert system.getParticleMass(5)._value == hydrogen_mass
+    assert system.getParticleMass(6)._value == hydrogen_mass
+    assert system.getParticleMass(7)._value == hydrogen_mass
```

Comment:
In the issue #1056 you note that "tests should make sure... Virtual sites continue to never have any mass". Should that be checked here?